src/main.py

```diff
@@ -33,28 +32,29 @@ def validate_config(cfg):
     logger.setLevel(logging.DEBUG)
     logger.info("starting ais-lattice-integration")
 
-    parser = argparse.ArgumentParser(description="AIS Vessel to Lattice Mesh Integration")
+    parser = argparse.ArgumentParser(
+        description="AIS Vessel to Lattice Mesh Integration"
+    )
     parser.add_argument(
-        "--config", action="store", dest="configpath", default="../var/config.yml"
+        "--config",
```

Comment:
Note, some of the changes here are due to the formatter kicking in on save. If it's an issue I can resubmit the PR without the formatting changes.